docs/src/3_Automation/05_Vulnerabilities.yml

```diff
@@ -1,140 +1,168 @@
-
-
-Template:           # This dictates the CSV columns and order thereof
-    Category:       # One string. If none, 'None'
-    Subcategory:    # One string. If none, 'None'
-    Type:           # Major, Minor, or GM Determined
-    Prereq Skill:
-    Prereq Power:
-    Tags:
-    # For longer items, if has ':', whole line must be quoted
-    # if multiple items, rest will appear as bullets after first in md
-    Description:
-    Mechanic:
+Template: # Power name        # Must be unique
+    Type: Vulny               # Vulny
+    Category: One or list     # One string or list in sub-cat order
+    XP: X                     # [-1,-2]
+    Prereq:                   # For multiple, implies 'OR'. Use [A, B] or `-`
+        Role: X
+        Level: X              # Integer               
+        Skill: X > 1          #              
+        Power:                # Must match unique power name
+    Description: Descrip      # Flavor: 1-2 sentences             
+    Tags: One or list
+    # Rest can be ignored when copy/pasting. Only needed so code can combine
+    PP: Ignore
+    To Hit: Ignore
+    Damage: Ignore
+    Range: Ignore
+    AOE: Ignore
+    Target: Ignore
+    Mechanic: Ignore
+    Save:
+        Trigger: Ignore
+        DR: Ignore
+        Type: Ignore
+        Fail: Ignore
+        Succeed: Ignore
 
 Bloodthirsty:
     Category: Combat
-    Subcategory: None
     Description: Your character is driven by a need for violence.
     Mechanic: Given the choice between a peaceful or violent solution, you must choose combat.
-    Type:   Minor
+    Type: Vulny
+    XP: -1
 
 Chaotic Companion:
     Category: Party
-    Subcategory: None
-    Description: You and your companion don’t always get along, sometimes they don’t do exactly what you ask.
-    Mechanic: When you draw a card to command your companion, the DM gains control of
-    Type: Minor or Major
+    Description: You and your companion don not always get along, sometimes they don not do exactly what you ask.
```

Comment:
Oh, oops. I removed apostrophes here because one of us saved using a different apostrophe in a different character encoding. UTF-8 is fine, others might work, but some will cause problems. I can go thru and fix contractions

Reply:
OK no worries, I noticed they were removed but from a formatting perspective I'm impartial to which encoding we opt for to render these correctly.